books/books.json

```diff
@@ -147,5 +147,22 @@
     "amazonASIN": "1118385349",
     "isbn": "978-1118385340",
     "publishedDate": "2014-10-10"
+  },
+  {
+    "id": "definitiveguide2modernjavaclients",
+    "summary": "This book is a reference for building Java applications for desktop, mobile and embedded.",
+    "publisher": "Apress",
+    "authors": "Stefen Chin, Johan Vos and James Weaver",
+    "personIds": [
+      "s.chin",
+      "johanvos",
```

Comment:
I think Johan's ID is j.vos, isn't it?

Reply:
You are right, I think I copied his Twitter instead of his id. My bad! :see_no_evil: